### PR TITLE
avatar and button

### PR DESCRIPTION
**Pull request by Shailseen**
Made avatar and button component. Please review it.

### CODE REVIEW

File: doc-css/nav.css [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+    align-items: center;
     gap: 2rem;
 }
 .contact-links{
     display: flex;
     flex-wrap: wrap;
     width: 100%;
     justify-content: flex-end;
-    border: 2px solid red;
+    /* border: 2px solid red; */
```
Comment (Aditi0505) [inline doc-css/nav.css]: Suggestion: Remove unnecessary comments. This is in multiple places.

File: docs.html [hunk 2]
```diff
@@ -53,6 +53,129 @@ <h4>Component</h4>
...
+            <h3>Avatar Examples</h3>
+            <p>Avatar available in four different sizes.
+                avtar can be add using claas <span class="highlight">avtar</span> and then add type of class name for different sizes are <span class="highlight">avtar-x-large</span> <span class="highlight">avtar-large</span> <span class="highlight">avtar-small</span> <span class="highlight">avtar-x-small</span> 
+            </p>
+            <h4>Example for adding class name</h4>
+            <span class="highlight">class = "avtar avtar-small"</span>
+          </div>
+          <div class="component-outputs">
+                <div class="avtar-outputs">
+                    <img class="avtar avtar-x-large" src="images/img_avatar.png">
```
Comment (Aditi0505) [inline docs.html]: Suggestion: Add alt attribute for accessibility.

File: docs.html [hunk 3]
```diff
@@ -53,6 +53,129 @@ <h4>Component</h4>
...
+                    <img class="avtar avtar-large" src="images/img_avatar.png">
+                    <img class="avtar avtar-small" src="images/img_avatar.png">
+                    <img class="avtar avtar-x-small" src="images/img_avatar.png">
+                </div>
+          </div>
+          <div class="code-snippet">
+            <code>
+              <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=sharp&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522avtar%2520avtar-x-large%2522%2520src%253D%2522images%252Fimg_avatar.png%2522%253E%250A%253Cimg%2520class%253D%2522avtar%2520avtar-large%2522%2520src%253D%2522images%252Fimg_avatar.png%2522%253E%250A%253Cimg%2520class%253D%2522avtar%2520avtar-small%2522%2520src%253D%2522images%252Fimg_avatar.png%2522%253E%250A%253Cimg%2520class%253D%2522avtar%2520avtar-x-small%2522%2520src%253D%2522images%252Fimg_avatar.png%2522%253E"
+              style=border:0; transform: scale(1); overflow:hidden;"
```
Comment (Aditi0505) [inline docs.html]: Suggestion: Can remove inline style and create a class for the same.

**Review by Aditi0505 — APPROVED** [review]
I have suggested some small changes. Rest looks good.

**Comment by TheAmlanSaikia** [comment]
All the things mentioned by Aditi and also use only single unit wherever possible, rest all looks good.

**Review by vishalg8454 — APPROVED** [review]
All fine, looks good to me now.

**Pull request merged**